intervals.md

```diff
@@ -1 +0,0 @@
-# intervals
```

Comment:
WRONG GROUP in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message